src/components/file-uploader/FileUploader.stories.mdx

```diff
@@ -0,0 +1,41 @@
+import { Meta, Story } from "@storybook/addon-docs/blocks";
+import FileUploader from "./FileUploader.vue";
+
+<Meta title="FileUploader" component={FileUploader} />
+
+# FileUploader
+
+Component which stores blob image data from user selected images and/or PDFs.
+
+## When to Use
+
+Use for getting the blob data of user selected images or PDFs. 
+
+## When not to Use
+
+Don't use as file uploader for non-images or non-PDF.
+
+## API
+
+Note: Requires the following NPM packages:
+```bash
+npm i blueimp-load-image --save
+npm i pdfjs-dist --save
+npm i rxjs --save
+npm i sha1 --save
+npm i uuid --save
+```
+
+| Prop        | Type          | Required | Description                                                         | Default |
+| ----------- | ------------- | -------- | ------------------------------------------------------------------- | ------- |
+| v-model     | Array         | true     | Stores the collection of images selected by the user.               |         |
+| instructionText | String    | false    | Subtitle text to display in the component.           .              | Please upload your documents. |
```

Comment:
table alignment? also in the description I see an extra `.` which does not seem to need to be there